packages/authentication/README.md

```diff
@@ -78,10 +87,14 @@ export class MyAuthStrategyProvider implements Provider<Strategy | undefined> {
     }
   }
 
-  verify(username: string, password: string, cb: Function) {
+  verify(
+    username: string,
+    password: string,
+    cb: (err: Error | null, user?: UserProfile | false) => void,
+  ) {
     // find user by name & password
     // call cb(null, false) when user not found
-    // call cb(null, userProfile) when user is authenticated
+    // call cb(null, user) when user is authenticated
```

Comment:
I think `user` should be `UserProfile` since that's the type.

Reply:
I wanted to avoid the callback type looking something like `cb: (err: Error | null, userProfile?: UserProfile | false) => void`, but maybe it'd ok here?

Reply:
On second thought, `user` is fine.